### PR TITLE
Fix a few Goodreads-related typos

### PR DESCRIPTION
**Pull request by Frenzie**

### CODE REVIEW

File: plugins/goodreads.koplugin/doublekeyvaluepage.lua [hunk 1]
```diff
@@ -176,7 +176,7 @@ local DoubleKeyValuePage = InputContainer:new{
     show_page = 1,
     text_input = "",
     pages = 1,
-    goodreadersKey = "",
```
Comment (houqp) [inline plugins/goodreads.koplugin/doublekeyvaluepage.lua]: Oops, I missed this part. I know you did not write this line, but could you rewrite this variable name to `goodreads_key`?

File: plugins/goodreads.koplugin/doublekeyvaluepage.lua [hunk 2]
```diff
@@ -195,11 +195,11 @@ function DoubleKeyValuePage:init()
     self.screen_height = Screen:getSize().h
     local gr_sett = self:readGRSettings().data
     if gr_sett.goodreads then
-        self.goodreadersKey = gr_sett.goodreads.key
-        self.goodreadersSecret = gr_sett.goodreads.secret
+        self.goodreadsKey = gr_sett.goodreads.key
+        self.goodreadsSecret = gr_sett.goodreads.secret
```
Comment (houqp) [inline plugins/goodreads.koplugin/doublekeyvaluepage.lua]: Same as my previous comment, should be `goodreads_secret`.

File: plugins/goodreads.koplugin/doublekeyvaluepage.lua (hunk 1 above)
Comment (Frenzie) [inline plugins/goodreads.koplugin/doublekeyvaluepage.lua]: Um, wait a second, both of your comments are requesting the change that's already been made! =P

Comment (Frenzie) [inline plugins/goodreads.koplugin/doublekeyvaluepage.lua]: Never mind, I just realized you're talking about case, not goodreaders.

**Comment by houqp** [comment]
All builds are failing right now due to new rules added to a new version of luacheck. Which is a good thing ;)

The luacheck fix is in https://github.com/koreader/koreader/pull/2341.

**Pull request merged**